staging/src/k8s.io/cloud-provider/controllers/service/controller_test.go

```diff
@@ -897,8 +916,22 @@ func compareUpdateCalls(t *testing.T, left, right []fakecloud.UpdateBalancerCall
 	}
 }
 
+func compareHostSets(t *testing.T, left, right []*v1.Node) bool {
```

Comment:
I don't understand this - it claims to compare sets, but it returns true as long as any one of left matches one of right - am I misreading it?

Reply:
Indeed, if I force a test case where the last node differs it still passes.

Reply:
Sorry, rectified this.